kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_empty_extension_property.bpmn

```diff
@@ -24,7 +24,7 @@
       <bpmn:incoming>SequenceFlow_02trahh</bpmn:incoming>
     </bpmn:endEvent>
     <bpmn:sequenceFlow id="SequenceFlow_02trahh" sourceRef="UserTask_1adz61s" targetRef="EndEvent_1rp66xw" />
-    <bpmn:serviceTask id="ServiceTask_13eimrx" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="ServiceTask_13eimrx" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```

Comment:
io.kadai.impl.ComplexProcessVariableSetter ?

Reply:
You're right, I will change it